### PR TITLE
rtmidi: 5.0.0

### PR DESCRIPTION
**Pull request by MartinDelille**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by MartinDelille** [comment]
I add a comment here: https://github.com/thestk/rtmidi/issues/155#issuecomment-994850151

**Review by SSE4 — APPROVED** [review]
LGTM

**Review by uilianries — APPROVED** [review]

File: recipes/rtmidi/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        self.cpp_info.names["pkg_config"] = "rtmidi"
```
Comment (prince-chrismc) [inline recipes/rtmidi/all/conanfile.py:78]: Missing new properties for v2 🤷

**Review by prince-chrismc — APPROVED** [review]

File: recipes/rtmidi/all/conanfile.py (hunk 1 above)
Comment (MartinDelille) [reply recipes/rtmidi/all/conanfile.py:78]: Ok I don't know exactly how to fix this but I started here: https://github.com/conan-io/conan-center-index/pull/9644

**Pull request merged**